staging/src/k8s.io/api/resource/v1/types.go

```diff
@@ -199,7 +205,7 @@ type CounterSet struct {
 	// Counters defines the set of counters for this CounterSet
 	// The name of each counter must be unique in that set and must be a DNS label.
 	//
-	// The maximum number of counters in all sets is 32.
+	// The maximum number of counters is 32.
 	//
 	// +required
 	// +k8s:required
```

Comment:
(for follow-up, once https://github.com/jpbetz/validation-gen/issues/208 is done we can add `// +k8s:maxProperties=32` here and in `DeviceCounterConsumption`)